app/pages/lab/export-workflows-dialog.jsx

```diff
@@ -90,7 +105,50 @@ class ExportWorkflowsDialog extends React.Component {
 ExportWorkflowsDialog.propTypes = {
   project: React.PropTypes.shape({ links: React.PropTypes.object }).isRequired,
   onSuccess: React.PropTypes.func.isRequired,
-  onFail: React.PropTypes.func.isRequired,
+  onFail: React.PropTypes.func.isRequired
+};
+
+const ExportWorkflowListItem = ({ workflow, media, onChange }) => {
+  const myMedia = workflow ? media[workflow.id.toString()] : null;
+  const now = new Date();
+  const lockoutTime = new Date();
+  lockoutTime.setDate(now.getDate() - 1);
+
+  const lockout = myMedia && (new Date(myMedia.updated_at) > lockoutTime);
+  const titleString = lockout ? 'This item can only be exported every 24 hours' : '';
+
+  return (
+    <li>
+      <input type="radio" title={titleString} name="which-workflow" id={`export-${workflow.id}`} disabled={lockout} onChange={onChange} />
+      {workflow.display_name}
+      <small>
+        <ExportWorkflowLink media={myMedia} />
+      </small>
+    </li>
+  );
+};
+
+ExportWorkflowListItem.propTypes = {
+  workflow: React.PropTypes.shape({ id: React.PropTypes.string, display_name: React.PropTypes.string }).isRequired,
+  media: React.PropTypes.shape({}).isRequired,
```

Comment:
I'd rather have it defined as an object then and add an inline linter rule exception rather than define it as a shape since that just looks like a mistake.